cleaned type-o in gitlab "admin.clientSecretDescription" references from "gitab" to "gitlab"

johncoleman83 opened the pull request:
#### Summary
As I was learning mattermost codebase, I found an error in which `gitlab` was referenced as `gitab` with no `l` (L).  Although I could not find a **[Help Wanted]** ticket for this, I wanted to contribute to get me started on this project.  Later, I would like to contribute more significantly when I better understand the mattermost codebase, and hope to be able to contribute to a [Help Wanted] Ticket in the issues section.  I think mattermost is a great system as it uses `go`, `react` & `redux`, and I hope to become a more significant contributor!

#### Ticket Link
I could not find this error (maybe it isn't an error?) in the issues tab.

#### Checklist
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates
- [x] Touches critical sections of the codebase (auth, posting, etc.)

jasonblais commented:
Also giving PM approval for accepting this pull request without a help wanted ticket. Justification: Fixes a typo in a string ID, with no effect on functionality.

@johncoleman83 Hope to see you around, welcome to the community! If you have interest on specific types of tickets (e.g. tickets on Go or React), let me know and we might have a few introductory tickets for you to try out.

Also, if you haven't yet, would love to see you join our public community instance where both core committers and community members discuss various topics daily: https://pre-release.mattermost.com/

johncoleman83 commented:
@jasonblais, hi!  Thanks for the review.  I mistakenly included file `block`.  I removed it from the PR.  I also removed all updates from all `json` files in `i18n`.

Thanks for the welcome!  I will soon sign-up and join the public community instance.  And I will reach out to you for guidance on an introductory ticket.  Thank you sincerely for offering the help.

enahum commented:
@johncoleman83 @jasonblais normally we include the changes only in the en.json file as the rest of the files we'll be updated with whatever translation was set in the translation server with the PR, so if included those changes will be lost really as they won't do anything.
